src/lazy/text/matched.rs

```diff
@@ -443,7 +443,7 @@ impl MatchedString {
     }
 }
 
-fn decode_text_containing_escapes(
+fn replace_escapes_with_byte_values(
```

Comment:
I like this name choice. Much better than the previous one and any of my suggestions.